msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting.md

```diff
@@ -53,6 +53,100 @@ The following table provides the user types and lists the features that each use
 | Federated or External | Available | Not available | Not available | Available |
 | Anonymous | Not available | Not available | Not available | Not available |
 
+## Targeted in-meeting notification
+
+Targeted in-meeting notification allows apps to send notifications to specific participants in a meeting. The notifications are private and are sent only to specific or targeted participants. Targeted in-meeting notification helps to enhance meeting experience and develop user engagement activities in Teams meetings.
+
+> [!NOTE]
+> Targeted in-meeting notification is supported for scheduled meetings, instant meeting (Meet now), one-on-one calls, and group calls.
+
+In the following image, a meeting notification requesting payment is sent to one of the participants in the meeting. The meeting notification is only visible to the targeted participant:
+
+ :::image type="content" source="../assets/images/apps-in-meetings/reminder-sent.png" alt-text="Screenshot shows an example of a meeting notification sent to a targeted participant requesting for a payment. ":::
+
+### Enable app manifest settings for targeted in-meeting notification
+
+To send targeted in-meeting notifications, you must configure the `authorization` property and the `name` and `type` properties under the `resourceSpecific` field in the [app manifest](../resources/schema/manifest-schema.md#authorization) as follows:
+
+```json
+"authorization": {
+    "permissions": {
+      "orgWide": [],
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeetingNotification.Send.Chat",
+          "type": "Application"
+        }
+      ]
+    }
+  }
+```
+
+### Enable targeted in-meeting notification
+
+> [!NOTE]
+> You can send a targeted in-meeting notification to 50 participants in a meeting. If you want to send a targeted in-meeting notification to more than 50 participants, you must call the `targetedMeetingNotification` API again.
```

Comment:
The first sentence "You can send a targeted in-meeting notification to 50 participants in a meeting" is disproven by the second sentence. You can send a notification to more than 50 people you just need to send multiple calls.


Can we change the first sentence to something like "You can only send a targeted in-meeting notification to 50 meetings participants in a single API call"

Reply:
Updated.